src/plugins/charts/bar/BarGraphView.vue

```diff
@@ -236,7 +236,32 @@ export default {
         refreshData(bounds, isTick) {
             if (!isTick) {
                 const telemetryObjects = Object.values(this.telemetryObjects);
-                telemetryObjects.forEach(this.requestDataFor);
+                telemetryObjects.forEach((telemetryObject) => {
+                    //clear existing data
+                    const key = this.openmct.objects.makeKeyString(telemetryObject.identifier);
+                    const axisMetadata = this.getAxisMetadata(telemetryObject);
+                    let trace = {
```

Comment:
This trace looks like it's copied/pasted from below. Can we generalize this into a method?